src/components/AccountDrawer/SettingsMenu.tsx

```diff
@@ -48,9 +49,11 @@ const ThemeToggleContainer = styled.div`
   margin: 0 0 6px;
 `
 
-const BalanceToggleContainer = styled.div`
-  padding: 16px 0;
-  margin-bottom: 26px;
+const ToggleContainer = styled.div`
+  padding: 16px 0 0 0;
```

Comment:
feels like this is used as a gap between the different toggle options right? maybe put a `gap` property on the parent div instead, and have `margin-bottom` be on the parent only